src/config/config.go

```diff
@@ -0,0 +1,124 @@
+package config
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v3"
+	"io/ioutil"
+	"net/url"
+	"os"
+)
+
+// Config struct holds configuration of Device Manager
+type Config struct {
+	Host               string   `yaml:"Host"`
+	Port               string   `yaml:"Port"`
+	UserName           string   `yaml:"UserName"`
+	Password           string   `yaml:"Password"`
+	RootServiceUUID    string   `yaml:"RootServiceUUID"`
+	OdimURL            string   `yaml:"OdimURL"`
+	OdimUserName       string   `yaml:"OdimUserName"`
+	OdimPassword       string   `yaml:"OdimPassword"`
+	TLSConf            *TLSConf `yaml:"TLSConf"`
+	RSAPrivateKeyPath  string   `yaml:"RSAPrivateKeyPath"`
```

Comment:
Are those two values(RSAPrivateKeyPath and RSAPublicKeyPath) needed?

Reply:
Updated config.